### ISSUE TITLE
$jscomp.inherits is not a function - Issues with jscomp object re-declared in each file when using require.js

### ISSUE THREAD

**Issue by sam6321** [google/closure-compiler#2398]
I've been having an issue with closure-compiler when used in conjunction with require.js. I've built a [minimal example](https://github.com/sam6321/closure-compiler_issue) that shows the problem.

Due to the way require.js loads modules, the $jscomp object is being declared in test1.js where it has the 'inherits' method, is then overwritten in test2.js where the 'inherits' method is no longer required.

When require.js loads the module in test1.js, the 'inherits' method no longer exists, causing the "$jscomp.inherits is not a function" error.

I'm using the following script to compile the js files:
```
#!/bin/bash
# Compiles all of the javascript in-place using closure-compiler

JS=$(find ./ -iname '*.js')
for i in $JS
do
    closure-compiler --warning_level QUIET --js $i --js_output_file $i.build --language_out ES5
    sudo mv $i.build $i
done
```
closure-compiler --version:
```
Closure Compiler (http://github.com/google/closure-compiler)
Version: v20170218
Built on: 2017-02-23 11:19
```
require.js version: RequireJS 2.3.3

**Comment by ChadKillingsworth** [comment]
I'm guessing this was fixed by f17feaad

**Comment by sam6321** [comment]
Ah, it would be. Thank you.
